flink-cdc-cli/src/main/java/org/apache/flink/cdc/cli/CliFrontendOptions.java

```diff
@@ -46,6 +46,16 @@ public class CliFrontendOptions {
                     .desc("JARs to be submitted together with the pipeline")
                     .build();
 
+    public static final Option TARGET =
+            Option.builder("t")
+                    .longOpt("target")
+                    .hasArg()
+                    .desc(
+                            "The deployment target for the execution. This can take one of the following values "
```

Comment:
Could we add user documentation about the new submit mode?

Reply:
IMO It may be better to add this part of doc in this PR, when supporting yarn mode, we can add the second part doc.

Reply:
https://issues.apache.org/jira/browse/FLINK-34677 Maybe can in there, WDHY?

Reply:
+1 for adding related docs in this PR since FLINK-34677 has added a new section `deployment/kubernetes` for Kubernetes deployment.